.github/workflows/release.yml

```
@@ -18,32 +18,40 @@ jobs:
-            View Release: ${{ github.event.release.html_url }}
-
-            **Changelog:**
-
-            ${{ github.event.release.body }}
+      # Print out the commit SHA
+      - name: Print commit SHA
+        run: echo "Commit SHA ${{ steps.release_action.outputs.commit_long_sha }}"
+
+      # Update preview tag using API
+      - name: Update preview tag
+        run: |
```

actionlint (1.7.7)

property "release_action" is not defined in object type {}

(expression)